app/views/pages/home.html.erb

```diff
@@ -102,9 +89,8 @@
         <h5 class="center">Nurturing Space</h5>
 
         <p class="light">
-          Our active community of military and veterans open source software &amp; hardware enthusiasts help you gain
-          real-world app experience, share job opportunities, conference scholarships, and opportunities to build on your
-          military experience.
+          Our growing community of software enthusiasts will help you learn about code schools, scholarships,
+          jobs, gain real-world programming experience, and more.
```

Comment:
What do you think about 'careers' instead of 'jobs?'

Also, 'gain real-world programming experince by contributing to open source?'